### DISCUSSION TITLE
Max Concurrent Jobs Option

### DISCUSSION THREAD

**Discussion by itizarsa** [jobrunr/jobrunr#1021] [General]
I read #13, is there a way to configure this now other than the way suggested in that issue?

Currently, I couldn't find a way to pass the concurrent option too.

What will be the max concurrent jobs if we don't provide the option?

**Comment by itizarsa** [comment]
I believe the `workerCount` is the maximum number of concurrent jobs allowed at any moment.

**Reply by auloin** [reply]
There is an example [here](https://www.jobrunr.io/en/documentation/configuration/fluent/#advanced-configuration):
```
JobRunr.configure()
// ...
            .useBackgroundJobServerIf(isBackgroundJobServerEnabled, 
                usingStandardBackgroundJobServerConfiguration().andWorkerCount(4))
// ...
```

or simpler:
```
JobRunr.configure()
// ...
            .useBackgroundJobServer(4)
// ...
```

see [JobRunrConfiguration](https://github.com/jobrunr/jobrunr/blob/master/core/src/main/java/org/jobrunr/configuration/JobRunrConfiguration.java) for more.

> I believe the `workerCount` is the maximum number of concurrent jobs allowed at any moment.

A more flexible approach is to use [rate limiters](https://www.jobrunr.io/en/documentation/pro/rate-limiters/), available in JobRunr Pro.

**Reply by itizarsa** [reply]
Thanks for the info
